srcpkgs/spnavcfg/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'spnavcfg'
+pkgname=spnavcfg
+version=1.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config qt5-host-tools"
+makedepends="libspnav-devel qt5-devel"
+depends="libspnav qt5-core qt5-gui qt5-widgets"
+short_desc="Spacenav interactive configuration GUI"
+maintainer="Alexandru Barbur <alex@ctrlc.name>"
+license="GPL-3.0-or-later"
+homepage="http://spacenav.sourceforge.net"
+changelog="https://github.com/FreeSpacenav/spnavcfg/releases"
+distfiles="https://github.com/FreeSpacenav/spnavcfg/releases/download/v${version}/${pkgname}-${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/FreeSpacenav/spnavcfg/releases/download/v${version}/spnavcfg-${version}.tar.gz"
```

Reply:
Fixed in https://github.com/void-linux/void-packages/commit/e462fc35ebda39fe2cf10d63b5423f1d03d4a0a0. I went back and forth on this and found examples of packages which do both haha. I'm guessing the rule here is to only template the `version` part?